addons/stock_account/models/account_move.py

```diff
@@ -132,35 +132,11 @@ def _stock_account_prepare_anglo_saxon_out_lines_vals(self):
                 amount_currency = sign * line.quantity * price_unit
 
                 # Add interim account line.
-                lines_vals_list.append({
-                    'name': line.name[:64],
-                    'move_id': move.id,
-                    'partner_id': move.commercial_partner_id.id,
-                    'product_id': line.product_id.id,
-                    'product_uom_id': line.product_uom_id.id,
-                    'quantity': line.quantity,
-                    'price_unit': price_unit,
-                    'amount_currency': -amount_currency,
-                    'account_id': debit_interim_account.id,
-                    'display_type': 'cogs',
-                    'tax_ids': [],
-                })
+                lines_vals_list.append(line._stock_account_prepare_anglo_saxon_interim_line_val(amount_currency, price_unit, debit_interim_account))
```

Comment:
The fact that you chose these arguments arbitrarily proves that this is probably not a good solution.
Why don't you add the product or the move_id for instance?